config.py

```diff
@@ -34,5 +29,9 @@
 channel_map = ""
 broken_channels = []
 view_offset = []
+view_chan_repet = []
 
+drift_length = 0.
+anode_z = 0.
+view_length = []
```

Comment:
Do we need to initialise these here with nonsense values? I think it should be sufficient that they are added as attributes to `cf` in `det_spec.py`.

Not sure whether that changes anything for importing them from other modules though...